deployment/terraform/assets/cluster.tf

```diff
@@ -593,7 +593,7 @@ resource "aws_security_group" "redis" {
     security_groups = [aws_security_group.app[0].id, aws_security_group.metrics[0].id]
   }
 
-  count = 1
+  count = var.app_instance_count > 0 && var.redis_enabled ? 1 : 0
```

Comment:
Do we need to check that there are app nodes deployed? Isn't it enough with checking that Redis is enabled?

Reply:
Agree. We should be able to use Redis even with single app node.

Reply:
I agree too, but both security groups in the array are tied to if you have created an app node.

Reply:
Aaaah, you're right! This is basically for the app to be able to communicate with Redis, so I guess it makes sense

Reply:
I am blind. I mistook `> 0` to be `> 1`. We generally don't test with no app nodes. Thanks for looking into these edge casees.